test/k6/src/tests/instances.js

```diff
@@ -73,12 +77,17 @@ export function setup() {
 
 // TC01 - POST instance
 function TC01_PostInstance(data) {
+  const options = {
+    "personNumber": data.personNumber
+  };
+
   var res = instancesApi.postInstance(
-    data.userToken,
+    data.orgToken,
```

Comment:
no need to change this to org token. The user is authorized to instantiate on behalf of themselves.

Reply:
Correct, as this is for the person number test.